benchmark/parse_output.py

```diff
@@ -0,0 +1,163 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015--, The WGS-HGT Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+#
+# Parse output files of HGT tools.
+#
+
+import click
+import sys
+
+
+def parse_trex(input_f):
+    """ Parse output of T-REX version 3.6.
+
+    Parameters
+    ----------
+    input_f: string
+        file descriptor for T-REX output results
+
+    Returns
+    -------
+    number_of_hgts: string
+        number of HGTs reported by a tool, or NaN if an error occurred
+    """
+    string = 'hgt : number of HGT(s) found = '
```

Comment:
OK, then what about creating a general function with this behavior and simply change the string based on tool/functions?